html/components/_buttons.scss

```diff
@@ -5,7 +5,8 @@
 ////
 /// @group button
 ////
-.sprk-c-Button {
+.sprk-c-Button,
+.sprk-c-Button.sprk-c-Button--loading[disabled] {
```

Comment:
this is done because we are incorrectly styled based on a disabled attribute. So I have to be more specific instead of undoing everything. 

We've all agreed to not style based on attribute or elements but haven't gotten around to doing it here. I'm making an issue to do for later because this would be a breaking change otherwise.